app/src/main/java/edu/wisc/ece/pinpoint/data/OrderedHashSet.java

```diff
@@ -0,0 +1,90 @@
+package edu.wisc.ece.pinpoint.data;
```

Comment:
I say we stick with this implementation for now and maybe switch to the TreeSet once we allow pin deletion? This implementation will be more efficient (constant time rather than log time) for adds & lookups, which will be the vast majority of our workload.